mne/io/eeglab/eeglab.py

```diff
@@ -39,8 +41,19 @@ def _check_mat_struct(fname):
         raise RuntimeError('scipy >= 0.12 must be installed for reading EEGLAB'
                            ' files.')
     from scipy import io
-    mat = io.whosmat(fname, struct_as_record=False,
-                     squeeze_me=True)
+    try:
+        # Try to read old style Matlab file
+        mat = io.whosmat(fname, struct_as_record=False, squeeze_me=True)
+    except:
```

Comment:
never `except:` always `except Exception` (the former will catch `SystemExit`). In fact it should be as explicit as possible. Can it be `TypeError`, `KeyError`, `ValueError`, or `IOError`?